src/services/vehicleDragger.ts

```diff
@@ -84,25 +112,33 @@ const enum DragState
 	Cancel
 }
 
+interface DragPosition
```

Comment:
Maybe this can be merged with `DragVehicleArgs` as well with reducing all duplicate fields?

Reply:
right. mb.

Reply:
it is now embedded within `DragVehicleArgs`. I still needed `DragPosition` to be stand alone to keep away from adding alot of new code to `getPositionFromTool`